client/guidestours/index.js

```diff
@@ -1,125 +1,20 @@
 /**
  * External dependencies
  */
-import React, { Component, PropTypes } from 'react'
+import React, { Component } from 'react'
```

Comment:
I think we'll probably converge to `guided-tours`, and I agree with moving to `layout`. That should however happen in its own PR, after this and pending ones get merged.